src/models/branches/lines.jl

```diff
@@ -12,8 +12,15 @@ struct Line <: Branch
     b::From_To_Float # [pu]
     rate::Float64
     anglelimits::Min_Max #Degrees
+    internal::PowerSystemInternal
 end
 
+function Line(name, available, connectionpoints, r, x, b, rate, anglelimits::Min_Max)
```

Comment:
Why is the last argument here alone typed?

Reply:
Looks like a bug

Reply:
Oversights. Thanks for catching them.  Fixed.